widgetssdk/src/main/java/com/glia/widgets/entrywidget/adapter/EntryWidgetAdapter.kt

```diff
@@ -55,12 +61,14 @@ internal class EntryWidgetAdapter(
     }
 
     enum class ViewType {
-        CONTACT_ITEM,
+        AUTHENTICATED_CONTACT_ITEM,
+        UNAUTHENTICATED_CONTACT_ITEM,
```

Comment:
Could you please explain what `AUTHENTICATED_CONTACT` and `UNAUTHENTICATED_CONTACT` stand for?

Reply:
`UNAUTHENTICATED_CONTACT ` - contact items (media types) that don't require visitor authentication: video, audio, live chat.
`AUTHENTICATED_CONTACT` - contact item (media type) that requires visitor authentication: secure messaging.

I can add this as comments to the code.
Let me know if you have better names in mind.

Reply:
IMHO, the reason we separated these items was the `Messaging type` with its unread count, so separating them by authentication status is a bit confusing.
I would prefer to have separated `live` items together and `messaging` smth like

- `LIVE_MEDIA_TYPE_ITEMS`
- `MESSAGING_MEDIA_TYPE_ITEM`

Reply:
Here is how the `ViewType` enum looked like initially:
```
enum class ViewType {
        CONTACT_ITEM,
        ERROR_ITEM,
        PROVIDED_BY_ITEM
}
```
Maybe it makes sense to keep the 'contact' name and change it as follows:
```
enum class ViewType {
        LIVE_CONTACT_ITEM,
        MESSAGING_CONTACT_ITEM
        ERROR_ITEM,
        PROVIDED_BY_ITEM
}
```
What do you think regarding this suggestion?

Reply:
To be honest, `Contact` really confuses me.
In my opinion, it doesn't reflect the items it handles.

Reply:
I'm okay with `LIVE_MEDIA_TYPE_ITEMS` and `MESSAGING_MEDIA_TYPE_ITEM`.
Let's go with it then. 👌

Reply:
Renamed this along with view holder, binding, and XML files 4a13b5cc6514aa646996536da88faf33ea49950f